google-ads/src/test/java/com/google/ads/googleads/lib/GoogleAdsClientTest.java

```diff
@@ -195,6 +195,28 @@ public void testBuildFromPropertiesFile_withoutLoginCustomerId() throws IOExcept
     assertGoogleAdsClient(client, null, true);
   }
 
+  /** Tests setting the login customer ID after the GoogleAdsClient is already built succeeds. */
+  @Test
+  public void testSetLoginCustomerIdAfterCreatingClient() throws IOException {
+    // Create a properties file in the temporary folder.
+    File propertiesFile = folder.newFile("ads.properties");
+    try (FileWriter propertiesFileWriter = new FileWriter(propertiesFile)) {
+      testProperties.store(propertiesFileWriter, null);
+    }
+
+    // Build a new client from the file.
+    GoogleAdsClient client =
+        GoogleAdsClient.newBuilder()
+            .fromPropertiesFile(propertiesFile)
+            .setTransportChannelProvider(localChannelProvider)
+            .build();
+    // Set a custom loginCustomerId on the previously created GoogleAdsClient.
```

Comment:
nit on here and line 198: Technically we're not setting this on the same GoogleAdsClient, but instead are creating a new builder from that instance, setting the login customer ID, and building a new instance.